packages/atlas/src/components/_navigation/SidenavBase/SidenavBase.styles.ts

```diff
@@ -1,40 +1,36 @@
-import isPropValid from '@emotion/is-prop-valid'
 import styled from '@emotion/styled'
-import { Link, LinkProps } from 'react-router-dom'
+import { Link } from 'react-router-dom'
 
-import { smallBadgeStyles } from '@/components/Badge'
 import { HamburgerButton } from '@/components/_buttons/HamburgerButton'
-import { cVar, media, oldColors, sizes, transitions, zIndex } from '@/styles'
-
-export const EXPANDED_SIDENAVBAR_WIDTH = 360
-export const NAVBAR_LEFT_PADDING = 24
+import {
+  EXPANDED_SIDENAVBAR_WIDTH,
+  EXPANDED_SIDENAVBAR_WIDTH_XXS,
+  NAVBAR_LEFT_PADDING,
+  SubItemProps,
+} from '@/components/_navigation/NavItem'
+import { cVar, media, sizes, transitions, zIndex } from '@/styles'
 
 type ExpandableElementProps = {
   expanded?: boolean
 }
 
-type SubItemProps = {
-  subitemsHeight?: number
-} & ExpandableElementProps
-
-type SidebarNavLinkProps = {
-  content: string
-} & ExpandableElementProps &
-  LinkProps
-
 export const SidebarNav = styled.nav<ExpandableElementProps>`
   position: fixed;
   top: 0;
   bottom: 0;
   height: 100%;
   z-index: ${zIndex.sideNav};
-  width: ${({ expanded }) => (expanded ? `${EXPANDED_SIDENAVBAR_WIDTH}px` : 'var(--size-sidenav-width-collapsed)')};
+  width: ${({ expanded }) => (expanded ? `${EXPANDED_SIDENAVBAR_WIDTH_XXS}px` : 'var(--size-sidenav-width-collapsed)')};
+  ${media.md} {
+    width: ${({ expanded }) => (expanded ? `${EXPANDED_SIDENAVBAR_WIDTH}px` : 'var(--size-sidenav-width-collapsed)')};
```

Comment:
I suggest we use some css variable for expanded size as well to avoid duplicated media queries. For example you define:

```
  --size-sidenav-width-expanded: 320px;
  ${media.md} {
    --size-sidenav-width-expanded: 360px;
  }
```

And then wherever width is used you just use `var(--size-sidenav-width-expanded)` without additional media query

Reply:
You can do that right above the lines I commented on